LIPs/LIP-1.md

```diff
@@ -0,0 +1,88 @@
+decision-making processes used in the governance around future Livepeer protocol upgrades and tools/environments used in Livepeer development.
+
+### LIP Statuses
+
+- **Draft**: a LIP that is open for consideration.
+- **Accepted**: a LIP that is planned for immediate adoption (i.e. expected to be included in an upcoming protocol upgrade).
+- **Final**: a LIP that has been adopted in a previous protocol upgrade.
+- **Deferred**: a LIP that is not being considered for immediate adoption, but may be reconsidered in the future for a subsequent protocol upgrade.
+- **Rejected**: a LIP that is not being considered for immediate adoption, and will not be reconsidered in the future for a subsequent protocol upgrade.
+
+### LIP Work Flow
+
+The LIP repository editors change the status of LIPs.
+
+The LIP process begins with a new idea for the Livepeer protocol. LIPs should be as focused as possible such that they contain a single key proposal or new idea.
+The LIP editor reserves the right to reject LIP proposals if they appear to be unfocused or too broad.
+
+Each LIP must have a champion - someone who writes the LIP using the appropriate style and format, shepherds discussions in the appropriate forums, and attempts to build
+community consensus around the proposed idea.
+
+Parties interested in submitting a LIP are encouraged to vet an idea publicly before actually writing the proposal to save time and to ensure that the idea presented is
+applicable to the entire Livepeer community. Examples of appropriate public forums to gauge interest around an LIP include [the Livepeer subreddit](https://www.reddit.com/r/livepeer), [the issues for this repository](https://github.com/livepeer/LIPs/issues), [the Livepeer Discord channels](https://discord.gg/7wRSUGX) and [the Livepeer forum](https://forum.livepeer.org/).
+
+After receiving feedback from the community and refining the technical language around an idea, a draft LIP should be submitted as [pull request](https://github.com/livepeer/LIPs/pulls).
+
+If the LIP editors approve the LIP and the author is happy with the draft being merged, the LIP editors will assign the LIP a number and merge the LIP as a draft.
+
+Once a draft LIP is merged, additional changes to the draft may be submitted as pull requests until the author believes the LIP is mature enough for the next phase.
+If the type of the LIP is informational or meta, the LIP can be designated as "Final" if there are no remaining technical objections. Some LIPs (such as this one) are meant to be
+continually updated in the future and will remain in the "Draft" stage. If the type of the LIP is standard track, the LIP is presented and discussed at a core developer meeting.
+
+If the participants of the core developer meeting do not voice any technical objections, the LIP is designated as "Accepted" and planned for immediate adoption. If there
```

Comment:
Does "Accepted" imply an intent to implement asap?

Reply:
Yep - I added additional clarification in c9bc5f8142af2be33023eca9c042a81e8efdbc16